src/afl-skip-log.cc

```diff
@@ -0,0 +1,54 @@
+#include <cstdio>
+#include <fstream>
+
+extern "C" {
+
+#include "afl-fuzz.h"
+
+}
+
+FILE *profile_fd;
+
+void plot_profile_data(afl_state_t *afl, struct queue_entry *q) {
+
+  if (!profile_fd) {
+
+    std::string out_dir = (const char *)afl->out_dir;
```

Comment:
Why does this need to be a c++ file? I see no point for that. and please merge it into afl-fuzz-stats.c